web/src/main/java/org/springframework/security/web/access/DefaultWebInvocationPrivilegeEvaluator.java

```diff
@@ -91,14 +90,8 @@ public boolean isAllowed(String contextPath, String uri, String method, Authenti
 		if (authentication == null) {
 			return false;
 		}
-		try {
-			this.securityInterceptor.getAccessDecisionManager().decide(authentication, filterInvocation, attributes);
-			return true;
-		}
-		catch (AccessDeniedException ex) {
-			logger.debug(LogMessage.format("%s denied for %s", filterInvocation, authentication), ex);
```

Comment:
Is it still possible to preserve the log here? Or, is there a reason that the log is no longer needed?

Reply:
Yes, I'll use the `verify` method instead. I'm also wondering if we should now move the `null` checks for `authentication` and `attributes` to the `AuthorizationManagerAdapter`, since the `attributes` can be `null` when the `AuthorizationManager` is configured using the `access(AuthorizationManager)` method.

Checks like this are already done in `AbstractSecurityInterceptor`, it might be worth combining them in `AuthorizationManagerAdapter`.

Reply:
I see that you changed to the `verify` method, as you explained. I'm wondering if it would still work to use `check` and then verify if the response `isDenied`? It's nice to avoid creating exceptions where we can since it's expensive.

Reply:
Sorry, I misunderstood you about logging, I thought that the exception should still be visible in the logs.

Reply:
@jzheaux It might be worth considering adding a default method 
`boolean isGranted(Supplier<Authentication> authentication, T object)` to the `AuthorizationManager`. What do you think?